### ISSUE TITLE
FormatException: Could not find an option named "packages"

### ISSUE THREAD

**Issue by Anrock** [dart-lang/coverage#123]
So in order to format coverage, readme suggests to use command `format_coverage --packages=<package_dir>/.packages -i <coverage json>`

Running this command with coverage-0.7.8 instantly produces
```
Unhandled exception:
FormatException: Could not find an option named "packages".
#0      Parser.validate (package:args/src/parser.dart:247)
#1      Parser.parseLongOption (package:args/src/parser.dart:236)
#2      Parser.parse (package:args/src/parser.dart:77)
#3      ArgParser.parse (package:args/src/arg_parser.dart:133)
#4      parseArgs (http://localhost:51721/format_coverage.dart:115)
#5      main.<main_async_body> (http://localhost:51721/format_coverage.dart:28)
#6      Future.Future.microtask.<anonymous closure> (dart:async/future.dart:144)
#7      _microtaskLoop (dart:async/schedule_microtask.dart:41)
#8      _startMicrotaskLoop (dart:async/schedule_microtask.dart:50)
#9      _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
#10     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:149)
```

What is proper command to produce human-readable coverage report?

**Comment by cbracken** [comment]
Are you sure you're using version 0.7.8? The stack trace you have suggests the version of `format_coverage` you're running doesn't have that [option](https://github.com/dart-lang/coverage/blob/master/bin/format_coverage.dart#L93), and the call site for `parseArgs` doesn't match the line number for version 0.7.8.

Your stack trace shows the call to `parseArgs()` at format_coverage.dart:115, but the call site as of version 0.7.8 is at [line 120](https://github.com/dart-lang/coverage/blob/master/bin/format_coverage.dart#L120). Line 115 was the [line number](https://github.com/dart-lang/coverage/blob/fa27a7a6e57f1433b8327767874c6d498dcd7dff/bin/format_coverage.dart#L115) for the call at version 0.7.7. 

Out of paranoia, I grabbed the published tarballs for 0.7.8, 0.7.8+1, 0.7.8+2 and all three appear to have the call site at line 120.

Can you do two things:
  1. Take a look at your `pubspec.yaml` in and verify it says 0.7.8.
  2. If you originally installed using `pub global activate`, can you pub global deactivate then `pub global activate` again?

**Comment by Anrock** [comment]
Oops, sorry. Just checked - `pub global activate coverage` actually installed 0.7.7 version. Deactivating and activating it installs 0.7.7 anyway.
